internal/download/get.go

```diff
@@ -45,8 +45,12 @@ func httpGet(url string) ([]byte, *failures.Failure) {
 }
 
 func httpGetWithProgress(url string, progress *progress.Progress) ([]byte, *failures.Failure) {
-	logging.Debug("Retrieving url: %s", url)
-	client := retryhttp.NewClient(0 /* 0 = no timeout */, 5)
+	return httpGetWithProgressRetry(url, progress, 3)
```

Comment:
I believe that this will prevent any tries. There are two bits of information needed here, I think. `attempt` and `max`.

Reply:
D'oh, good eye.